icu4c/source/i18n/measunit.cpp

```diff
@@ -2061,6 +2075,8 @@ MeasureUnit &MeasureUnit::operator=(MeasureUnit &&other) noexcept {
     }
     delete fImpl;
     fImpl = other.fImpl;
+    // FIXME(review): Here it is again, looks to the naive eye like "destruction
```

Comment:
Yeah, this is move assignment.

```
MeasureUnit foo(MeasureUnit::getSecond()); // move constructor
foo = MeasureUnit::getMeter(); // move assignment
```

Reply:
#TIL about move assignment operators. :-D